sortingAlgorithms.js

```diff
@@ -9,26 +9,28 @@
 // [1][2][5]
 
 function bubblesort(array, testFunction) {
+  const myArray = array;
+  // let OptionalTestFunction = testFunction;
   if (!testFunction) {
-    testFunction = function (a, b) {
+    testFunction = function regularSort(a, b) { // TypeError: testFunction is not a function
       return a - b;
     };
   }
-  for (let j = 0; array.length > j; j++) {
-    for (let i = array.length - 1; i > 0; i--) {
-      let leftElement = array[i - 1];
-      let rightElement = array[i];
+  for (let j = 0; myArray.length > j; j++) {
```

Comment:
Naming: "myArray" ist kein guter Name

Reply:
Habe ich jetzt in "ArrayToSort" umbenannt.